lua/notagain/essential/autorun/server/mount_gmas.lua

```diff
@@ -0,0 +1,13 @@
+local path = "notagain/gmafiles/"
+local gma_path = "addons/notagain/lua/"..path
```

Comment:
the PAC3 server merges notagain with lua root directory, so this would work in singleplayer, but not on our server.